src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const listImage = document.querySelector('ul');
+
+listImage.addEventListener('click', (item) => {
+  item.preventDefault();
```

Comment:
Ensure that `item` is indeed an event object. The `preventDefault` method should be called on an event object to prevent the default action of the event from occurring.